### PR TITLE
Improve data dictionary script usage

### PR DESCRIPTION
**Pull request by mcelestini**
- add a scripts readme
- add a command to make file to build the data dictionary
- improve the script

### CODE REVIEW

File: docs/scripts.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**_Build Data Dictionary_**
+
+To build `dataDictionary.json`, paste a tab delimited .txt file into the root of the repo and run `make build-dictionary`
```
Comment (SchaeStewart) [inline docs/scripts.md:3]: Can you add an example header with the fields you expect. Also document the default filename, and how to pass a custom file name if possible

Comment (mcelestini) [reply docs/scripts.md:3]: I'm actually not sure how to get command line params to work with a makefile. I tried following this SO post
https://stackoverflow.com/questions/6273608/how-to-pass-argument-to-makefile-from-command-line
with the following changes
```
build-dictionary:
	cd scripts && node createDataDictionary $(filter-out, $@, $(MAKECMDGOALS))
```
but no luck when I try something like
`make build-dictionary myFile.txt`

Comment (SchaeStewart) [reply docs/scripts.md:3]: Yeah, make doesn't really work well with arguments. For the makefile you can use a sane default. For the documentation, just show how to run the script manually/with a custom file path. Also we need to document what headers are expected in the file

**Review by SchaeStewart — CHANGES_REQUESTED** [review]

**Comment by mcelestini** [comment]
@SchaeStewart I think I've addressed everything, let me know if this is good to go and I can merge it 👍

**Review by SchaeStewart — APPROVED** [review]
Merge when ready

**Pull request merged**